content/tutorials/single-page-tutorials/proof-inclusion-of-a-block.md

```diff
@@ -0,0 +1,351 @@
+
+```javascript
+const {
+  SingleNodeClient,
+  LocalPowProvider,
+  sendData,
+} = require('@iota/iota.js');
+const fs = require('fs');
+const fetch = require('node-fetch');
+
+// Network configuration
+const { networkConfig } = require('./networkConfig.js');
+const nodeURL = networkConfig.node;
+const explorerURL = networkConfig.explorer;
+
+// For better readability, some console output will be printed in a different color
+const consoleColor = '\x1b[36m%s\x1b[0m';
+```
+
+#### 2. Main function
+The `main()` function will be called when running the `create-notarization.js` file and consists of the following steps.
+
+Declare the function and setup a node client for the interaction with the network.
+```javascript
+async function main() {
+  // Setup client and define block content
+  const client = new SingleNodeClient(nodeURL, {
+    powProvider: new LocalPowProvider(),
+  });
+```
+
+Define the content (`tag` & `data`), attach new block to the Tangle and log out the explorer link.
+```javascript
+  // Define block content, attach block to the Tangle and log out the explorer link
+  const tag = 'This is my Tag';
+  const data = 'This is my data';
+
+  const sendResult = await sendData(client, tag, data);
+  const blockId = sendResult.blockId;
+
+  console.log(consoleColor, 'Attached block:');
+  console.log(explorerURL + 'block/' + blockId, '\n');
+```
+
+Wait for block confirmation by a milestone and read the block with proof of inclusion from INX plugin. The function `getNotarization()` will be explained in more detail in the next section.
+```javascript
+  // Wait for block confirmation by milestone and read it with proof of inclusion from INX plugin
+  const result = await getNotarization(client, nodeURL, blockId);
+```
+
+This part will only be exectured, if the respective block was confirmed by a milestone after a defined time. If that is the case, the returned notarization result will be stored in a *.json file and the file path will be logged out.
+```javascript
+  // Store block with proof of inclusion in local json file
+  if (result != false) {
+    const filePath = `./notarized-block.json`;
+    fs.writeFileSync(filePath, JSON.stringify(result, null, 4));
+
+    console.log(
+      consoleColor,
+      'Block successfully notarized and stored at:'
+    );
+    console.log(filePath, '\n');
+
+    console.log(
+      consoleColor,
+      'The Notarized block can now be handed over to the verifier',
+    );
+  }
+}
+```
+
+#### 3. Get notarization
+As described in the previous step, the `getNotarization()` function is called from within `main()`. The function fetches the metadata of a given blockId 10 times while pausing for 1 second after each try via the `sleep()` function. As soon as a block was referenced by a milestone, the notarization for the block is fetched from the proof of inclusion plugin and  returned by the function. In case the block was not referenced by a milestone after 10 seconds, the function returns `false`.
```

Comment:
```suggestion
As described in the previous step, the `getNotarization()` function is called from within `main()`. As soon as a block was referenced by a milestone, the notarization for the block is fetched from the proof of inclusion plugin and returned by the function. The function tries to fetch the metadata of a given blockId 10 times, waiting for a second before each try. In case the block was still not referenced by a milestone, the function returns `false`.
```

I brought the more important information forward and rephrased it a bit.